e2e/test/scenarios/actions/actions-on-dashboards.cy.spec.js

```diff
@@ -56,16 +61,25 @@ const MODEL_NAME = "Test Action Model";
 
       describe("adding and executing actions", () => {
         beforeEach(() => {
+          resetSnowplow();
           resetTestTable({ type: dialect, table: TEST_TABLE });
           restore(`${dialect}-writable`);
           cy.signInAsAdmin();
+          enableTracking();
           resyncDatabase({ dbId: WRITABLE_DB_ID, tableName: TEST_TABLE });
           createModelFromTableName({
             tableName: TEST_TABLE,
             modelName: MODEL_NAME,
           });
         });
 
+        afterEach(() => {
+          expectNoBadSnowplowEvents();
+          expectGoodSnowplowEvent({
```

Comment:
Could you move `expectGoodSnowplowEvent` after the line that performs the action that submits the event? Currently it's a bit hard to follow